src/api/db/build.ts

```diff
@@ -282,3 +282,50 @@ export function getBuildStatus(buildId: number): Promise<any> {
       });
     });
 }
+
+export function getDepracatedBuilds(buildId: number): Promise<any> {
+  return new Promise((resolve, reject) => {
+    new Build({ id: buildId }).fetch()
+      .then(build => {
+        if (!build) {
+          reject(build);
+        }
+        build = build.toJSON();
+
+        new Build({ repositories_id: build.repositories_id })
```

Comment:
is there a reason for running two queries on a `builds` table, you run the query few lines before that already.
let's be rational using resources.